sklearn/metrics/_ranking.py

```diff
@@ -968,6 +968,75 @@ def roc_curve(y_true, y_score, *, pos_label=None, sample_weight=None,
 
     return fpr, tpr, thresholds
 
+def cumulative_gain_curve(y_true, y_score, pos_label=None):
+    """This function generates the points necessary to plot the Cumulative Gain for each ten percent of the samples
+    Note: This implementation is restricted to the binary classification task.
```

Comment:
Each line should be less than 80 char, and the first sentence of the docstring should be a 1 line summary.